data.md

```diff
@@ -230,16 +231,17 @@ The `#width` function returns the bit-width of a given `IValType`.
     rule #pow (i64) => 18446744073709551616
 ```
 
-Here we define the rules about hex int parsing.
+Here we define the rules about integer parsing.
 
 ```k
-    syntax HexWord
-    syntax String ::= #parseHexWordString ( HexWord ) [function, functional, hook(STRING.token2string)]
-    syntax Int    ::= #parseHex           ( String )  [function]
-                    | HexWord
+    syntax WasmInt
+    syntax String ::= #parseWasmIntToken ( WasmInt ) [function, functional, hook(STRING.token2string)]
+    syntax Int    ::= #parseWasmInt      ( String )  [function]
+                    | WasmInt
  // -------------------------
```

Comment:
We shouldn't be subsorting into `Int`, because it can break symbolic reasoning. Can we instead do something like:

```
syntax WasmInt ::= Int | OTHER_PRODUCTIONS
rule WS:WasmInt => #parseWasmInt(WS) requires notBool isInt(WS)
```